coverity.bat

```diff
@@ -0,0 +1,74 @@
+@ECHO OFF
+
+SETLOCAL
+
+PUSHD %~dp0
+
+IF NOT DEFINED COVDIR SET "COVDIR=C:\cov-analysis"
+IF DEFINED COVDIR IF NOT EXIST "%COVDIR%" (
+    ECHO.
+    ECHO ERROR: Coverity not found in "%COVDIR%"
+    GOTO End
+)
+
+
+CALL "%VS140COMNTOOLS%\vsvars32.bat"
+IF %ERRORLEVEL% NEQ 0 (
+    ECHO vsvars32.bat call failed.
+    GOTO End
+)
+
+
+:Cleanup
+IF EXIST "cov-int"  RD /q /s "cov-int"
+IF EXIST "gcw.lzma" DEL "gcw.lzma"
+IF EXIST "gcw.tar"  DEL "gcw.tar"
+IF EXIST "gcw.tgz"  DEL "gcw.tgz"
+
+
+:Main
+SET MSBUILD_SWITCHES=/nologo /t:Rebuild /p:Configuration=Release /p:Platform="Any CPU"^
+ /maxcpucount /consoleloggerparameters:DisableMPLogging;Summary;Verbosity=minimal
+
+"%COVDIR%\bin\cov-build.exe" --dir cov-int MSBuild.exe Microsoft.Alm.sln %MSBUILD_SWITCHES%
+
+
+:tar
+SET PATH=C:\MSYS\bin;%PATH%
```

Comment:
I should probably move this to the top. I use it on my rig in order to use `tar` from MSYS which results in a lot smaller archive.